README.md

```diff
@@ -91,3 +93,8 @@ See example documentation on:
 
 * [tests.sh](https://github.com/reconquest/tests.sh/blob/master/REFERENCE.md)
 * [coproc.bash](https://github.com/reconquest/coproc.bash/blob/master/REFERENCE.md)
```

Comment:
Those lines should not be in markdown block (starting from `Examples`).

Reply:
Moved ` ```` ` with comment about `@internal` tag upper, to have examples at the end and not escaped